packages/data-broker/src/__tests__/services/dhis/DhisService.test.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import * as GetDhisApiInstance from '../../../services/dhis/getDhisApiInstance';
+import { DATA_SOURCES, DATA_VALUES, DHIS_REFERENCE, SERVER_NAME } from './DhisService.fixtures';
+import { DhisService } from '../../../services/dhis';
+import { createModelsStub, stubDhisApi } from './DhisService.stubs';
+
+describe('DhisService', () => {
```

Comment:
Moving back in a single file has two benefits:
- Does not hide the fact that DhisService is doing a lot. If this test suite becomes too long that's a code smell that the class is doing too much and needs to be broken up. We have done a little with breaking out pullers to (IMO) good effect, we could in the future do the same for Pushers and Deleters.
- Lets us more easily reason about how test setups and teardowns interact. Previously you had to look at 3 separate files to check the initialisers of the DhisService under test and the mocks, and run the risk of having tests interfere with each other, now these are all inline.